src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -2016,6 +2016,21 @@ public void Dispose()
             oldTimer?.Dispose();
         }
 
+        /// <summary>
+        /// Release all resources associated with this object.
+        /// </summary>
+        public async ValueTask DisposeAsync()
+        {
+            GC.SuppressFinalize(this);
+            await CloseAsync(!_isDisposed);
+            if (sentinelConnection is ConnectionMultiplexer sentinel)
+            {
+                await (sentinel?.DisposeAsync() ?? default);
```

Comment:
```suggestion
                await sentinel.DisposeAsync();
```

Reply:
Tried to be clear in comment - the `is ConnectionMultiplexer` is doing the null check here - there's zero chance it's null here so let's keep it simple please!

Reply:
To add: I would fix this and commit but since the checkbox isn't enabled for us to push changes or PR, can only suggest here.

Reply:
You were perfectly clear. I just messed up the commit after testing the two solutions. I also thought you'd be able to push commits, but maybe it wasn't enabled because I created this first as a draft pull request. Sorry for the inconvenience.

Reply:
_Allow edits by maintainers_ is enabled. I don't know why it doesn't work.